app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -66,12 +68,25 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
       .catch(miqService.handleFailure);
   };
 
+  $scope.filterCloudNetworkChanged = function(id) {
+    miqService.sparkleOn();
+    $http.get('/network_router/network_router_subnets_by_network/' + id)
```

Comment:
@gildub I see that you have added a new controller route and action -- `network_router_subnets_by_network`

Currently we have a new practice in place for all our angular forms, which is -- 
Use REST API calls and avoid creating new controller actions, especially true for any new functionality being added in the forms.

Hence I recommend using an API call here.